ballsdex/packages/players/cog.py

```diff
@@ -449,9 +452,48 @@ async def give(
         countryball.favorite = False
         await countryball.save()
 
-        await interaction.response.send_message(
+        await interaction.followup.send(
             f"You just gave the {settings.collectible_name} "
             f"{countryball.description(short=True, include_emoji=True, bot=self.bot)} to "
             f"{user.mention}!"
         )
         del self.bot.locked_balls[countryball.id]
+
+    @balls.command(name="count")
+    @app_commands.checks.has_any_role(*settings.root_role_ids)
+    async def balls_count(
+        self,
+        interaction: discord.Interaction,
+        ball: BallEnabledTransform = None,
+        shiny: bool = None,
+        special: SpecialEnabledTransform = None,
+    ):
+        """
+        Count the number of balls that you have.
+
+        Parameters
+        ----------
+
+        ball: Ball
+            The ball you want to count.
+        shiny: bool
+            Whether the ball is shiny or not.
+        special: Special
+            The special background of the ball.
+        """
+        filters = {}
```

Comment:
It's going to query everything lol
```suggestion
        filters = {"player__discord_id": interaction.user.id}
```